sdk/core/azure-core/src/io/body_stream.cpp

```diff
@@ -98,8 +98,8 @@ FileBodyStream::FileBodyStream(const std::string& filename)
   HANDLE fileHandle = INVALID_HANDLE_VALUE;
   try
   {
-#if !defined(WINAPI_PARTITION_DESKTOP) \
-    || WINAPI_PARTITION_DESKTOP // See azure/core/platform.hpp for explanation.
+#if (!defined(WINAPI_PARTITION_DESKTOP) || WINAPI_PARTITION_DESKTOP) \
+    && (!defined(WINAPI_FAMILY_APP)) // See azure/core/platform.hpp for explanation.
```

Comment:
It is strange that you had to modify it. I guess if it compiles, it is ok, but just checking if this is an accidental change.

Reply:
nope, it is by design, the uwp build did not support the API there

Reply:
You must be mistaking. This is SDK code. We are compiling our SDK for `arm-uwp` and `arm64-uwp` for a year now, with the ifdef that was: `vcpkg install azure-core-cpp:arm-uwp azure-core-cpp:arm64-uwp`.

Reply:
according to the bug https://app.zenhub.com/workspaces/adp-native-team-5f68c995a9966e1e074f078d/issues/azure/azure-sdk-for-cpp/2945 it is not working , and has not been working for a while

Reply:
after discussing with Anton this PR is actually not needed, his PR will supersede this one